### PR TITLE
Adjust tailwind preset export

### PR DESCRIPTION
**Pull request by DSil**
The tailwind config preset can be simplified into just one, as it happens that **both** are always needed by our users.

Since both are always needed, we can export just the most complete one, but document only the foundation. Everything else available can be used at the users' own risk.

This is a breaking change, since we change the default export. But we're on an early stage of adoption, so the impact will be low.

I have also updated the documentation accordingly.
 Storybook: https://orbit-mainframev-adjust-tailwind-preset-export.surge.sh

### CODE REVIEW

File: packages/orbit-components/tailwind.config.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
-const { default: orbitPreset, orbitComponentsPreset } = require("@kiwicom/orbit-tailwind-preset");
+const orbitPreset = require("@kiwicom/orbit-tailwind-preset");
```
Comment (mainframev) [inline packages/orbit-components/tailwind.config.js:1]: can you refactor it to ts file in the same way as it's #3942 branch? 🙏🏻

**Comment by mainframev** [comment]
LGTM, just one suggestion :)

**Comment by mainframev** [comment]
also @DSil as soon it's as a breaking changes, let's maybe merge it to #3942 so it will be available with v10 release, wdyt?

**Comment by DSil** [comment]
The commit with the BC only affects the tailwind-preset package, so it doesn't make a difference where we merge it, as it will only affect that package anyway. Also, the time between releases, ideally, won't be too different, so I don't know if it's worth it. I'm ok with any decision though

**Pull request merged**